pkg/common/common.go

```diff
@@ -0,0 +1,245 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	configutils "github.com/keptn/go-utils/pkg/configuration-service/utils"
+
+	keptnutils "github.com/keptn/go-utils/pkg/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var RunLocal = (os.Getenv("env") == "runlocal")
```

Comment:
I'll change this to `local` and `localtest`, to be compatible with our other services.